### PR TITLE
Inline stats

### PR DESCRIPTION
**Pull request by phlax**
Render stats inline

### CODE REVIEW

File: pootle/templates/browser/_table.html [hunk 1]
```diff
@@ -34,74 +34,78 @@
...
-        <div id="translated-ratio-{{ item.code|cssid }}" class="sortkey"></div>
+	{% if item.stats.total %}
+        <div id="translated-ratio-{{ item.code|cssid }}" class="sortkey">
+	  {% widthratio item.stats.translated item.stats.total 1000 %}
+	</div>
         <div id="progressbar-{{ item.code|cssid }}">
-          {% include "browser/_progressbar.html" %}
+          {% progress_bar item.stats.total item.stats.fuzzy item.stats.translated %}
         </div>
+	{% endif %}
```
Comment (unho) [inline pootle/templates/browser/_table.html]: Nit: Some of the changed/new lines don't seem to be correctly indented.

File: pootle/templates/browser/index.html [hunk 2]
```diff
@@ -158,16 +164,14 @@ <h3 class="top">{% trans "Updates" %}</h3>
 {{ block.super }}
 
 <script type="text/javascript">
-$(function () {
-  PTL.search.init();
+  $(function () {
```
Comment (unho) [inline pootle/templates/browser/index.html]: Nit: This was indented correctly before.

Comment (phlax) [inline pootle/templates/browser/index.html]: i beg to differ - it should indent inside js tags

File: pootle/templates/browser/index.html [hunk 3]
```diff
@@ -158,16 +164,14 @@ <h3 class="top">{% trans "Updates" %}</h3>
...
+
   PTL.stats.init({
     pootlePath: "{{ pootle_path }}",
     hasDisabledItems: {{ table.disabled_items|yesno:"true,false" }},
     isAdmin: {{ has_admin_access|yesno:"true,false" }},
```
Comment (unho) [inline pootle/templates/browser/index.html]: I think this is one is unnecessary as well.

Comment (unho) [inline pootle/templates/browser/index.html]: Is `isAdmin` actually necessary?

**Comment by unho** [comment]
Testing exposed some minor issues (e.g. #5507) but I am gtm.

**Pull request merged**